ros2_generate_interface_docs/ros2_generate_interface_docs/main.py

```diff
@@ -32,11 +32,11 @@ def generate_interfaces_index(messages, services, actions, html_dir, timestamp):
     """
     Generate index for packages.
 
-    :param messages: Dictionay with the message package as a key and a list message names as value
+    :param messages: Message package as a key and a list message names as value
```

Comment:
Nitpick: the key is not a *message package* but a *package name*. (Not "message" since it can contain more than that, but "name" since the argument is not a package instance but a string.)